docker/README.md

```diff
@@ -0,0 +1,31 @@
+# Docker
+
+This directory contains additional Dockerfiles for images built by Aerie.
+
+- [Dockerfile.hasura](./Dockerfile.hasura) - A Hasura Docker image with bundled Aerie-specific Hasura metadata
+- [Dockerfile.postgres](./Dockerfile.postgres) - A Postgres Docker image with bundled Aerie-specific SQL
+
+## Build
+
+First build Aerie to make sure the SQL files are properly added to the [deployment](../deployment/) directory:
+
+```sh
+cd aerie
+./gradlew assemble
+```
+
+Next, still from the top-level Aerie directory, build the images from the provided Dockerfiles:
+
+```sh
+docker build -t aerie-hasura -f ./docker/Dockerfile.hasura .
+docker build -t aerie-postgres -f ./docker/Dockerfile.postgres .
+```
+
+## Run
+
+To run the images you can use the following commands. Note these are just for testing purposes:
```

Comment:
Regarding the "note these are just for testing purposes": Are the files in `docker` meant for us to use internally or are they meant to be used by users?